inject/src/main/java/io/micronaut/context/env/AbstractPropertySourceLoader.java

```diff
@@ -95,17 +100,17 @@ public int getOrder() {
     private Map<String, Object> loadProperties(ResourceLoader resourceLoader, String qualifiedName, String fileName) {
         Optional<InputStream> config = readInput(resourceLoader, fileName);
         if (config.isPresent()) {
-            if (LOG.isDebugEnabled()) {
-                LOG.debug("Found PropertySource for file name: " + fileName);
+            if (logEnabled) {
```

Comment:
Shouldn't this be:

```suggestion
            if (logEnabled && LOG.isDebugEnabled()) {
```

Same in the other places

Reply:
@alvarosanchez And here I would argue: calling `LOG.isDebugEnabled()` and other `isXXXEnabled` methods makes sense if you have some calculations before calling the logging method. If you just make a method call without calculation, like `log.debug("My message {}", myParam);`, it makes no sense for you to check the logging level, because methods debug / info / error etc alredy have it inside. By the way, this is one of the reasons why logging formatting was invented with symbols `{}`. in this place you haven't any calculation for logging and don't need to check log level twice